lab_tests/README.md

```diff
@@ -42,7 +42,7 @@ Other actions:
 To run the script using a specific file, do
 
 ```bash
-python parse_daily_tests.py --xlsx ./path/to/file.xlsx
+python parse_daily_tests.py --xlsx ./data/path/to/file.xlsx
```

Comment:
this is an example path, why add `/data` here? revert this change please and instead change the default in line 28 to  `/data`